nat-lab/tests/mesh_api.py

```diff
@@ -196,14 +222,24 @@ def get_meshmap(
 
         return meshmap
 
-    def default_config_one_node(self, is_local: bool = False) -> Node:
-        alpha, *_ = self.config_dynamic_nodes([is_local])
+    def default_config_one_node(
+        self,
+        is_local: bool = False,
+        ip_stack: IPStack = IPStack.IPv4,
+    ) -> Node:
+        alpha, *_ = self.config_dynamic_nodes([(is_local, ip_stack)])
         return alpha
 
     def default_config_two_nodes(
-        self, alpha_is_local: bool = False, beta_is_local: bool = False
+        self,
+        alpha_is_local: bool = False,
+        beta_is_local: bool = False,
+        alpha_ip_stack: IPStack = IPStack.IPv4,
+        beta_ip_stack: IPStack = IPStack.IPv4,
```

Comment:
I'm not sure if enabling IPv6 in the kernel and disabling its routing is the best default scenario, I guess using `IPStack.IPv4v6` here would be more intuitive.

Reply:
I left an IPv4 as a default option, because of all the legacy tests, because it is still our primary stack, until we migrate to IPv4v6/IPv6 stacks.

Reply:
And routing itself is not disabled, just `ip6tables` rules are not explicitly added, if IPv4 stack is used.

Reply:
>because of all the legacy tests
Yeah, but I guess 

>because it is still our primary stack
But what is the problem with having both?

Reply:
They should work, but our priority is still 100% working condition IPv4 stack, so I am leaning towards to leave it IPv4 as default, before we'll toggle the IPv6 switch in prod.